### PR TITLE
Move securitychecks into own workflow

### PR DESCRIPTION
**Pull request by 6543**
- [x] activate backend again
- [x] move

followups:
- make trivy server work
- create notifications

### CODE REVIEW

File: .woodpecker/securityscan.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    group: check
+    image: *trivy_plugin
+    settings:
+      skip-dirs: web/,docs/
+
+  check docs:
+    group: check
+    image: *trivy_plugin
+    settings:
+      skip-dirs: node_modules/,plugins/woodpecker-plugins/node_modules
```
Comment (lafriks) [inline .woodpecker/securityscan.yml:21]: For consistency
```suggestion
      skip-dirs: node_modules/,plugins/woodpecker-plugins/node_modules/
```

**Comment by pat-s** [comment]
Sufficient to only run on `main`?

**Comment by 6543** [comment]
well we can do that once we get notification if something is wrong

**Comment by pat-s** [comment]
I don't quite understand your reply. Can you rephrase? 😅 

My thinking was that a PR shouldn't introduce a sec risk usually - and these usually come by some existing old library in which an issue get's detected.
It's also a resources question: is it really needed to run this WF (for a possible minor gain) in every PR? Hence my proposal above.

**Comment by 6543** [comment]
I dont think 14sec are a burden

**Comment by 6543** [comment]
> Can you rephrase?

the idea of mine was, to let it only run as cron job and create a discord&matrix notification if it do detect something.
I don't want to do this atm as we would only spontaneously notice it.

But while reading your replay, I realized that we should still run on pulls too!

yes it's rare that there are sec issues newly introduced in pulls, but for <=15sec, it it worth to just scan for them.

being as secure as possible should be one of our goals.

It should just not block pulls, and this new workflow will do exactly that :)

File: .woodpecker/securityscan.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      skip-dirs: web/,docs/
+
+  check docs:
+    group: check
+    image: *trivy_plugin
+    settings:
+      skip-dirs: node_modules/,plugins/woodpecker-plugins/node_modules/
+      dir:  docs/
+      # TODO currently it is not fixable so just do not block currently
+      exit-code: 0
```
Comment (6543) [inline .woodpecker/securityscan.yml:24]:
```suggestion
```
